images/docker/cbdb/build/rocky8/Dockerfile

```diff
@@ -87,8 +87,8 @@ RUN dnf makecache && \
         diffutils \
         file \
         flex \
-        gcc-toolset-11-gcc \
-        gcc-toolset-11-gcc-c++ \
+        gcc \
```

Comment:
Please consider amending the commit message to include the default GCC version on Rocky Linux 8. I believe GCC 8.5.0 is used:

```
$ rpm -aq | grep gcc
gcc-8.5.0-26.el8_10.x86_64
gcc-c++-8.5.0-26.el8_10.x86_64
gcc-plugin-annobin-8.5.0-26.el8_10.x86_64
libgcc-8.5.0-26.el8_10.x86_64
```

Reply:
Thanks. Updated.